music21/romanText/tsvConverter.py

```diff
@@ -388,30 +420,45 @@ def __init__(self):
         self.dcml_version = 2
 
     @property
-    def beat(self):
+    def beat(self) -> float:
         '''
         'beat' has been removed from DCML v2 in favor of 'mn_onset' and
         'mc_onset'. 'mn_onset' is equivalent to 'beat', except that 'mn_onset'
-        is zero-indexed where 'beat' was 1-indexed. This property reproduces
-        the former 'beat' by adding 1 to 'mn_onset'.
+        is zero-indexed where 'beat' was 1-indexed, and 'mn_onset' is in
+        fractions of a whole-note rather than in quarter notes.
         >>> tabCd = romanText.tsvConverter.TabChordV2()
-        >>> tabCd.mn_onset = 0
+        >>> tabCd.mn_onset = 0.0
+        >>> tabCd.beat
+        1.0
+        >>> tabCd.mn_onset = 1/2
```

Comment:
write as 0.5 -- otherwise it will be mistaken as a Fraction class and loses the parallel with beat.